platoon/channel.py

```diff
@@ -230,6 +243,13 @@ def __init__(self, control_port, port=None, socket_timeout=300000, hwm=10):
         self._job_uid = self.send_req("platoon-get_job_uid")
         self._lock = posix_ipc.Semaphore("{}lock".format(self._job_uid))
 
+    def start_compilation(self):
+        while not self.send_req("platoon-start_compilation"):
+            time.sleep(1)
```

Comment:
This is mostly what the theano cache do, but via the file system... So mostly, the gain is bypassing the filesystem? Do you see something else?

In fact, I don't understand why it is slower with multiple process compiling and waiting in parallel then with one process. I think it would be good to understand that... It could be that we overload the filesystem.

On which cluster did you had problems with this?